torchtext/experimental/datasets/language_modeling.py

```diff
@@ -61,7 +61,7 @@ def get_vocab(self):
 
 
 def _setup_datasets(dataset_name, tokenizer=None, root='.data', vocab=None,
```

Comment:
I'd remove the default arguments and make everything positional. All callsites must provide values for all keywords, so removing the default arguments here should also improve the certainty that indeed all arguments were forwarded.

Reply:
Same applies to all other _setup_datasets functions